piplmesh/frontend/views.py

```
@@ -22,6 +28,33 @@ class OutsideView(generic_views.TemplateView):
 class SearchView(generic_views.TemplateView):
     template_name = 'search.html'
 
+class ContactView(edit_views.FormView):
+    """
+    This view checks if all contact data are valid and then send mail.
+    
+    User is redirected to home page.
+    """
+    
+    template_name = 'contact.html'
+    # TODO: Redirect users to the page they initially came from
+    success_url = urlresolvers.reverse_lazy('home')
+    form_class = forms.ContactForm
+
+    def form_valid(self, form):
+        subject = form.cleaned_data['subject'],
+        from_email = form.cleaned_data['email'],
+        message = form.cleaned_data['message'],
+        try:
+            send_mail(subject, message, from_email, (settings.DEFAULT_FROM_EMAIL,), fail_silently=False)
+        except:
```

Never ever catch all exceptions. Only catch those you expect. Only catch `smtplib.SMTPException` here.